kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/events/EventContext.kt

```diff
@@ -78,4 +82,30 @@ public open class EventContext<T : Event>(
         key: String,
         replacements: Array<Any?> = arrayOf()
     ): String = translate(key, eventHandler.extension.bundle, replacements)
+
+    /**
+     * Given a translation key and possible replacements,return the translation for the given locale in the
```

Comment:
Commas require spaces after.

Reply:
Fixed all occurences of this, including the original ones.